Puzzlepart/did
# UI improvements

### Your checklist for this pull request
 
### Review checklist

### Review theme song
_Remove section if you think this is stupid._
🎵  [Polo & Pan - Ani Kuni](https://open.spotify.com/track/12QphPukpWDR9w1iimddfP) 🎵 

### Description
UI improvements - more air, better readability, division of days. Less SAP, more GitHub

![image](https://user-images.githubusercontent.com/7300548/142373387-9b721954-07d7-4609-b826-e0e542fff2c7.png)

![image](https://user-images.githubusercontent.com/7300548/142373474-d76f43cb-4511-4c12-b1cd-be8ed6859a01.png)


### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Click around
3. Observe the UI
